Userland/Libraries/LibJS/JIT/Compiler.cpp

```diff
@@ -1194,6 +1239,64 @@ void Compiler::compile_get_by_id(Bytecode::Op::GetById const& op)
     branch_if_object(ARG1, [&] {
         extract_object_pointer(GPR0, ARG1);
 
+        // NOTE: Fast path for Array.length which magically reflects
+        //       the "array-like size" of the array object's property storage.
+        Assembler::Label no_magical_length_property_case;
```

Comment:
Would it be possible to move both this label and corresponding link inside the below if statement?

Reply:
Sure, that would clean up the outer scope a little bit. Good idea :)